apps/condo/domains/ticket/access/Ticket.js

```diff
@@ -87,18 +80,26 @@ async function canReadTickets ({ authentication: { item: user }, context }) {
     }
 }
 
-async function canManageTickets ({ authentication: { item: user }, operation, itemId, originalInput }) {
+async function canManageTickets ({ authentication: { item: user }, operation, itemId, originalInput, context }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
     if (user.isAdmin) return true
 
     if (user.type === RESIDENT) {
-        let unitName, unitType, propertyId
```

Comment:
should we check the fields? is the resident can create a Ticket with a property from the wrong organization? (probably not a par of this task)